web/packages/build/vite/config.ts

```diff
@@ -100,7 +100,7 @@ export function createViteConfig(
           changeOrigin: false,
           secure: false,
         },
-        '^\\/v1\\/webapi\\/assistant': {
```

Comment:
was this intended to go here?

Reply:
Yes, since I changed the WS API path, I also needed to change the routing path in Vite to match it.